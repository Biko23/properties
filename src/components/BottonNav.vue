<template>
     <div id="col-hidden">
      <v-app-bar color="#4477fb" dense dark>
        <v-btn value="recent" style="background-color:#4477fb;width">
          <img
            src="https://res.cloudinary.com/diued7ugb/image/upload/v1625949058/sms_black_24dp_3_og1r7y.png"
            alt=""
            width="30"
            srcset=""
            style=""
          />
          <span style="color:#ef9e9e;">14</span>

          <!-- <v-icon>mdi-history</v-icon> -->
        </v-btn>
        <v-btn style="background-color:#4477fb;">
          <img
            src="https://res.cloudinary.com/diued7ugb/image/upload/v1625949073/notifications_black_24dp_2_jvvady.png"
            alt=""
            width="30"
            srcset=""
            style=""
          />
          <span style="color:#ef9e9e;">14</span>

          <!-- <v-icon>mdi-history</v-icon> -->
        </v-btn>

        <v-btn style="background-color:#4477fb;">
          <img
            src="https://res.cloudinary.com/diued7ugb/image/upload/v1625948894/home_black_24dp_5_sng2rd.png"
            alt=""
            width="30"
            srcset=""
            style=""
          />

          <!-- <v-icon>mdi-history</v-icon> -->
        </v-btn>

        <v-btn value="recent" style="width: 80px; margin-left: auto;background-color:#4477fb;">
          <span>Buy/Rent</span>

          <!-- <v-icon>mdi-history</v-icon> -->
        </v-btn>

        <v-btn value="recent" style="width: 60px; margin-left: auto; background-color:#4477fb;">
          <span>Sell</span>

          <!-- <v-icon>mdi-history</v-icon> -->
        </v-btn>
      </v-app-bar>
    </div>
</template>
<script>
export default {
    name:'BottomNav'
}
</script>

<style scoped>
#main-div {
  background-color: #3b6ef3;
}
#form-row {
  background-color: #e7f0ff;
  width: 1000px;
  margin-left: auto;
  margin-right: auto;
  border-radius: 6px;
}
#description {
  margin-top: -200px;
}
.custom-placeholer-color input::placeholder {
  color: red !important;
  opacity: 1;
}

.custom-label-color .v-label .solo {
  color: blue;
  opacity: 1;
}
#col-hidden{
    display: none;
}
@media only screen and (max-width: 768px) {
  #main-div {
    background-color: #3b6ef3;
  }
  #form-row {
    background-color: #e7f0ff;
    width: auto;
    margin-left: auto;
    margin-right: auto;
    border-radius: 6px;
  }
  #description {
    margin-top: auto;
  }
  #col-hidden{
    display: block;
}
}
</style>