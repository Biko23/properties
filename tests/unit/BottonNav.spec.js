import { createLocalVue, mount } from '@vue/test-utils'
import Vuetify from 'vuetify'
import BottonNav from '@/components/BottonNav.vue'

describe('BottonNav.vue', () => {
  const localVue = createLocalVue()
  let vuetify

  beforeEach(() => {
    vuetify = new Vuetify()
  })

  const mountFunction = options => {
    return mount(BottonNav, {
      localVue,
      vuetify,
      ...options,
    })
  }

  it('should have a match snapshot', () => {
    const wrapper = mountFunction()
    expect(wrapper.html()).toMatchSnapshot()
  })
})

