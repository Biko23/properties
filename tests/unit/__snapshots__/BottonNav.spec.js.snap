// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottonNav.vue should have a match snapshot 1`] = `
<div id="col-hidden">
  <header class="v-sheet theme--dark v-toolbar v-toolbar--dense v-app-bar" style="height: 48px; margin-top: 0px; transform: translateY(0px); left: 0px; right: 0px; background-color: rgb(68, 119, 251); border-color: #4477fb;">
    <div class="v-toolbar__content" style="height: 48px;"><button type="button" value="recent" class="v-btn v-btn--is-elevated v-btn--has-bg theme--dark v-size--default" style="background-color: rgb(68, 119, 251);"><span class="v-btn__content"><img src="https://res.cloudinary.com/diued7ugb/image/upload/v1625949058/sms_black_24dp_3_og1r7y.png" alt="" width="30" srcset=""> <span style="color: rgb(239, 158, 158);">14</span></span></button> <button type="button" class="v-btn v-btn--is-elevated v-btn--has-bg theme--dark v-size--default" style="background-color: rgb(68, 119, 251);"><span class="v-btn__content"><img src="https://res.cloudinary.com/diued7ugb/image/upload/v1625949073/notifications_black_24dp_2_jvvady.png" alt="" width="30" srcset=""> <span style="color: rgb(239, 158, 158);">14</span></span></button> <button type="button" class="v-btn v-btn--is-elevated v-btn--has-bg theme--dark v-size--default" style="background-color: rgb(68, 119, 251);"><span class="v-btn__content"><img src="https://res.cloudinary.com/diued7ugb/image/upload/v1625948894/home_black_24dp_5_sng2rd.png" alt="" width="30" srcset=""></span></button> <button type="button" value="recent" class="v-btn v-btn--is-elevated v-btn--has-bg theme--dark v-size--default" style="width: 80px; margin-left: auto; background-color: rgb(68, 119, 251);"><span class="v-btn__content"><span>Buy/Rent</span></span></button> <button type="button" value="recent" class="v-btn v-btn--is-elevated v-btn--has-bg theme--dark v-size--default" style="width: 60px; margin-left: auto; background-color: rgb(68, 119, 251);"><span class="v-btn__content"><span>Sell</span></span></button></div>
  </header>
</div>
`;
